### PR TITLE
submit corrected data

### PR DESCRIPTION
**Pull request by kreier**
updated to second version

### CODE REVIEW

**Comment by kreier** [comment]
Please compare the changes in software and comment on them.

**Comment by aisvn-data** [comment]
I need to review and comment the changes.

File: software/SolarAISVN/SolarAISVN.ino [hunk 1]
```diff
@@ -25,14 +25,21 @@ uint64_t uS_TO_S_FACTOR = 1000000;  // Conversion factor for micro seconds to se
 uint64_t TIME_TO_SLEEP = 120;
 
 int adcValue = 0;
+int batValue = 0;
```
Comment (aisvn-data) [inline software/SolarAISVN/SolarAISVN.ino:28]: Finally you measure the voltage of the LiPo Battery as well!

File: software/SolarAISVN/SolarAISVN.ino [hunk 2]
```diff
@@ -25,14 +25,21 @@ uint64_t uS_TO_S_FACTOR = 1000000;  // Conversion factor for micro seconds to se
...
 
 void setup() {
+  pinMode(LED_BUILTIN, OUTPUT);
+  digitalWrite(LED_BUILTIN, LOW);  
   Serial.begin(115200); 
-  delay(2000);
+  delay(50);
+  digitalWrite(LED_BUILTIN, HIGH);   
+  delay(1000);
```
Comment (aisvn-data) [inline software/SolarAISVN/SolarAISVN.ino:36]: Just a short blink added, not 2 seconds of light

File: software/SolarAISVN/SolarAISVN.ino [hunk 3]
```diff
@@ -93,8 +100,9 @@ void makeIFTTTRequest() {
 
   // raw and converted voltage reading
   adcValue = analogRead( 34 );
-  String jsonObject = String("{\"value1\":\"") + adcValue + "\",\"value2\":\"" + (adcValue * 2.4)
-                      + "\",\"value3\":\"" + millis() + "\"}";
+  batValue = analogRead( 35 );
+  String jsonObject = String("{\"value1\":\"") + adcValue + "\",\"value2\":\"" + int((adcValue * 0.826 + 150) * 3)
+                      + "\",\"value3\":\"" + int((batValue * 0.826 + 150) * 2) + "\"}";
```
Comment (aisvn-data) [inline software/SolarAISVN/SolarAISVN.ino:105]: Submit the battery value as well.

File: software/SolarAISVN/SolarAISVN.ino (hunk 1 above)
Comment (kreier) [reply software/SolarAISVN/SolarAISVN.ino:28]: Thanks. Do you have the ohmic values for your voltage divider?

File: software/SolarAISVN/SolarAISVN.ino (hunk 3 above)
Comment (kreier) [reply software/SolarAISVN/SolarAISVN.ino:105]: Please relocate the calculation of the battery voltage value to a different location.

File: software/SolarAISVN/SolarAISVN.ino (hunk 2 above)
Comment (kreier) [reply software/SolarAISVN/SolarAISVN.ino:36]: Good optical feedback.

**Comment by kreier** [comment]
Ready to merge?

Your voltage values could be determined with this non straight
t-koala
 resistors ..

File: software/SolarAISVN/SolarAISVN.ino (hunk 1 above)
Comment (aisvn-data) [reply software/SolarAISVN/SolarAISVN.ino:28]: Look at this picture and the two resistors. Not parallel, but they divide 1:1

Value can be checked:
divider

**Comment by aisvn-data** [comment]
Resistors are equally 47kOhm to divide the voltage 2:1

**Pull request merged**